include_server/test_data/supported_encoding.cpp

```diff
@@ -0,0 +1,2 @@
+�
```

Comment:
Is this change needed?  It might just be the web interface on GitHub making it look a little strange to me.

Reply:
Probably... it is an "empty" file in ISO-8859 encoding (as reported by `file`).

Reply:
Yes, it is actually an empty file with ISO-8859 encoding used by the unit test